Towny/src/main/java/com/palmergames/bukkit/towny/event/TownPreClaimEvent.java

```diff
@@ -118,9 +131,17 @@ public Player getPlayer() {
     	return player;
     }
 
+	/**
+	 * Useful to get selections of a claim action. 
+	 *
+	 * @return the selections of this claim.
+	 */
+	public List<WorldCoord> getSelection() {return selection}
+
```

Comment:
```suggestion
	/**
	 * Returns the list of WorldCoords which will be claimed when there is more 
 	 * than one, or null when this event is thrown via a new town, or takeoverclaim scenario. 
	 *
	 * @return the selections of this claim, or null.
	 */
	@Nullable
	public List<WorldCoord> getSelection() {
			return selection
	}

```

This will requite the Nullable import.